src/queries/finops-hub-database-guide.md

```diff
@@ -0,0 +1,692 @@
+//
+// Savings
+| extend x_AmortizationCategory = case(
+    ChargeCategory == 'Purchase' and isnotempty(CommitmentDiscountCategory), 'Principal',
+    isnotempty(CommitmentDiscountCategory), 'Amortized Charge',
+    ''
+)
+| extend x_CommitmentDiscountSavings = iff(ContractedCost == 0,      decimal(0), ContractedCost - EffectiveCost)
+| extend x_NegotiatedDiscountSavings = iff(ListCost == 0,            decimal(0), ListCost - ContractedCost)
+| extend x_TotalSavings              = iff(ListCost == 0,            decimal(0), ListCost - EffectiveCost)
+| extend x_CommitmentDiscountPercent = iff(ContractedUnitPrice == 0, decimal(0), (ContractedUnitPrice - x_EffectiveUnitPrice) / ContractedUnitPrice)
+| extend x_NegotiatedDiscountPercent = iff(ListUnitPrice == 0,       decimal(0), (ListUnitPrice - ContractedUnitPrice) / ListUnitPrice)
+| extend x_TotalDiscountPercent      = iff(ListUnitPrice == 0,       decimal(0), (ListUnitPrice - x_EffectiveUnitPrice) / ListUnitPrice)
+//
+// Toolkit
+| extend x_ToolkitTool = tostring(Tags['ftk-tool'])
+| extend x_ToolkitVersion = tostring(Tags['ftk-version'])
+| extend tmp_ResourceParent = database('Ingestion').parse_resourceid(Tags['cm-resource-parent'])
+| extend x_ResourceParentId = tostring(tmp_ResourceParent.ResourceId)
+| extend x_ResourceParentName = tostring(tmp_ResourceParent.ResourceName)
+| extend x_ResourceParentType = tostring(tmp_ResourceParent.ResourceType)
+//
+// TODO: Only add differentiators when the name is not unique
+| extend CommitmentDiscountNameUnique = iff(isempty(CommitmentDiscountId), '', strcat(CommitmentDiscountName, ' (', CommitmentDiscountType, ')'))
+| extend ResourceNameUnique           = iff(isempty(ResourceId),           '', strcat(ResourceName,           ' (', ResourceType, ')'))
+| extend x_ResourceGroupNameUnique    = iff(isempty(x_ResourceGroupName),  '', strcat(x_ResourceGroupName,    ' (', SubAccountName, ')'))
+| extend SubAccountNameUnique         = iff(isempty(SubAccountId),         '', strcat(SubAccountName,         ' (', split(SubAccountId, '/')[3], ')'))
+//
+// Explain why cost is 0
+| extend x_FreeReason = case(
+    BilledCost != 0.0 or EffectiveCost != 0.0, '',
+    PricingCategory == 'Committed', strcat('Unknown ', CommitmentDiscountStatus, ' Commitment'),
+    x_BilledUnitPrice == 0.0 and x_EffectiveUnitPrice == 0.0 and ContractedUnitPrice == 0.0 and ListUnitPrice == 0.0 and isempty(CommitmentDiscountType), case(
+        x_SkuDescription contains 'Trial', 'Trial',
+        x_SkuDescription contains 'Preview', 'Preview',
+        'Other'
+    ),
+    x_BilledUnitPrice > 0.0 or x_EffectiveUnitPrice > 0.0, case(
+        PricingQuantity > 0.0, 'Low Usage',
+        PricingQuantity == 0.0, 'No Usage',
+        'Unknown Negative Quantity'
+    ),
+    'Unknown'
+)
+//
+| project-away tmp_SQLAHB, tmp_IsVMUsage, tmp_ResourceParent
+```
+
+---
+
+## Table Reference
```

Comment:
I was really hoping to avoid having this. But if we're going to have it, shouldn't it be in the public docs rather than here?